bodhi/tests/server/services/test_comments.py

```diff
@@ -521,7 +521,7 @@ def test_list_comments_with_unexisting_ignore_user(self):
         self.assertEqual(len(body.get('comments', [])), 0)
         self.assertEqual(res.json_body['errors'][0]['name'], 'ignore_user')
         self.assertEqual(res.json_body['errors'][0]['description'],
-                         "Invalid user specified: santa")
+                         "Invalid users specified: santa")
 
     def test_put_json_comment(self):
         """ We only want to POST comments, not PUT. """
```

Comment:
Let's add some tests that assert that the ```update_owner```, ```ignore_user```, and ```user``` parameters can accept multiple values.